contrib/test.sh

```diff
@@ -13,6 +14,11 @@ if cargo --version | grep nightly >/dev/null; then
     NIGHTLY=true
 fi
 
+if cargo --version | grep ${MSRV}; then
+    # memcrh 2.6.0 uses edition 2021
```

Comment:
fix this, like in https://github.com/rust-bitcoin/hex-conservative/pull/33/files